package.json

```diff
@@ -36,15 +37,19 @@
     "typescript": "^3.5.2"
   },
   "devDependencies": {
-    "@types/cors": "^2.8.5",
-    "@types/glob": "^7.1.1",
     "@types/chai-http": "^4.2.0",
```

Comment:
Remove also stuff related to the libraries you don't use any more since you removed the old tests 👍